src/utils/cssRuleHasSelectorEndingWithColon.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Check whether a rule has a selector ending in a colon
+ *
+ * @param {Rule} postcss rule node
+ * @return {boolean} If `true`, the rule has a selector ending in a colon
+ */
+export default function (rule) {
+  const selector = rule.raws.selector && rule.raws.selector.raw || rule.selector
```

Comment:
I like cleaning up this kind of thing with lodash's `_.get()`: `const selector = _.get(rule, 'raws.selector.raw', rule.selector)` But that's up to you, just an idea.